kiva/quartz/ABCGI.pyx

```diff
@@ -1591,7 +1591,7 @@ cdef class CGBitmapContext(CGContext):
         if file_format is None:
             file_format = ''
 
-        img = pilfromstring(mode, (self.width(), self.height()), self)
+        img = PilImage.frombytes(mode, (self.width(), self.height()), self)
```

Comment:
This is interesting. `Image.frombytes` is necessary here because this class doesn't hold it's data in a numpy array, but _does_ support the [legacy] [buffer interface](https://cython.readthedocs.io/en/latest/src/userguide/special_methods.html#buffer-interface-legacy-no-python-equivalents-see-note-1). That said, `Image.frombuffer` should also work, and it would avoid a copy.